posts/crud_post.py

```diff
@@ -5,6 +5,8 @@
 from db_utils import get_db
 from auth import auth
 from util import have
+import uuid0
```

Comment:
You aren't using the correct import order

https://www.python.org/dev/peps/pep-0008/#imports
Correct order:
- Standard library imports.
- Related third party imports.
- Local application/library specific imports.